.env.example

```diff
@@ -0,0 +1,8 @@
+# Local development variables. When developped locally with Saleor inside docker, these can be set to:
+#
+# APP_IFRAME_BASE_URL = http://localhost:3000, so Dashboard on host can access iframe
+# APP_API_BASE_URL=http://host.docker.internal:3000 - so Saleor can reach App running on host, from the container.
+#
+# If developped with tunnels, set this empty, it will fallback to default Next's localhost:3000
```

Comment:
Not quite true. When using tunnels, it will use tunnel URL instead of localhost. Information is taken from the request.

Maybe simplification would be - it depends on requested URL. If you enter with "localhost/api/manifest" it will be localhost, in case of tunnel user use "XXXX.ngrok.com/api/manifest".

Also, there's a typo:
```suggestion
# If developed with tunnels, set this empty. It will fallback to default Next's localhost:3000
```

Reply:
TBH description from this file could be taken as is and put into `docs/development.md`.
You are describing the same thing here and in factory. Don't think it has to be perfect on the first try, but reading it in one file would be much easier than gathering info from two places

Reply:
We have (and will have one more) docs on apps development in https://docs.saleor.io

Inline docs like these are meant to be something else - you browse code and you have inline hints what is this + link do longer document.

Putting this to docs/development.md will be duplicating this https://docs.saleor.io/docs/3.x/developer/app-store/development

Rel content - will fix

Reply:
Fully agree!